src/NewCard/primitives/CardElement.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import { css } from "styled-components";
+
+import { getBorder } from "./borders";
+
+// eslint-disable-next-line import/prefer-default-export
+export const CardElement = css`
```

Comment:
This is more mixin than primitive, consider creating `helpers` folder as we do it for all other components, e.g. Stack.

Reply:
@vepor i know that it's mixin, but it can be there in my opinion

Reply:
Stick to common folder structure that we have please 🙏